emp/emp_role_cftemplate.yaml

```diff
@@ -0,0 +1,370 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Parameters:
+  RoleName:
+    Type: String
+    Description: The name of the IAM role and instance profile.
+    Default: 'pf9emp'
+
+Resources:
+  # CAPA Policies
+  AWSIAMManagedPolicyCloudProviderNodes:
+    Properties:
+      Description: For the Kubernetes Cloud Provider AWS nodes
+      ManagedPolicyName: !Sub "${RoleName}-nodes.cluster-api-provider-aws.sigs.k8s.io"
+      PolicyDocument:
+        Statement:
+          - Action:
+              - ec2:AssignIpv6Addresses
+              - ec2:DescribeInstances
+              - ec2:DescribeRegions
+              - ec2:CreateTags
+              - ec2:DescribeTags
+              - ec2:DescribeNetworkInterfaces
+#              - ec2:DescribeInstanceTypes commenting, not deleting capa actions. This is accounted for below
+              # check with team if ecr:<action is needed>
+              - ecr:GetAuthorizationToken
+              - ecr:BatchCheckLayerAvailability
+              - ecr:GetDownloadUrlForLayer
+              - ecr:GetRepositoryPolicy
+              - ecr:DescribeRepositories
+              - ecr:ListImages
+              - ecr:BatchGetImage
+            Effect: Allow
+            Resource:
+              - '*'
+          - Action:
+              # capa creates encrypted cloud init secrets, this is needed.
+              - secretsmanager:DeleteSecret
+              - secretsmanager:GetSecretValue
```

Comment:
We disable secrets manager. would this still be needed?

Reply:
yes, we're going to re-enable them after 1.1